[Inductor][Observability] Change to log.debug to avoid excessive long of logs

mengluy0125 opened the pull request:
Summary: Titled

Test Plan: CI

Differential Revision: D52003825




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

Skylion007 commented on torch/_inductor/select_algorithm.py:
```diff
@@ -724,7 +724,7 @@ def __call__(
                 "config (defined in torch/_inductor/config.py) to allow at least one choice. "
             )
-        log.info("Max autotune selects from %s choices.", str(len(choices)))
+        log.debug("Max autotune selects from %s choices.", str(len(choices)))
```
```suggestion
        log.debug("Max autotune selects from %d choices.", len(choices))
```

mengluy0125 committed:
[Inductor][Observability] Change to log.debug to avoid excessive long of logs (#115474)

Summary:

Titled

Test Plan: CI

Reviewed By: xuzhao9, yanboliang, jackiexu1992

Differential Revision: D52003825

oulgen commented on torch/_inductor/compile_fx.py:
```diff
@@ -510,7 +510,7 @@ def fx_codegen_and_compile(
         post_grad_passes(gm, is_inference=is_inference)
         V.debug.fx_graph_transformed(gm, example_inputs)
-        post_grad_graphs_log.info("%s", lazy_format_graph_code("AFTER POST GRAD", gm))
+        post_grad_graphs_log.debug("%s", lazy_format_graph_code("AFTER POST GRAD", gm))
```
why was this one changed? this is the only thing post_grad_graphs log logs
